apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -499,28 +490,29 @@ export class WorkshopForm extends React.Component {
           <Col sm={5}>
             <FormGroup validationState={validation.style.suppress_email}>
               <ControlLabel>
-                Enable email notifications?
+                Enable workshop reminder notifications?
                 <HelpTip>
                   <p>
-                    Code.org can send email notifications about this workshop to
-                    your attendees on your behalf. Notifications may include:
+                    <strong>
+                      This functionality is disabled for all academic year
+                      workshops and virtual CSF workshops.
```

Comment:
should we give a reason for this here? I worry we will get a lot of questions with just a blanket "disabled" message.

Reply:
I went back and forth with Tawny on the wording for tooltips a bunch for these, I agree maybe a little vague but trust she knows partners better than I do.

Reply:
sounds good 👍